### DISCUSSION TITLE
An enhancement to the quote system

### DISCUSSION THREAD

**Discussion by cris16228** [Docile-Alligator/Infinity-For-Reddit#868] [Ideas]
I would like to suggest and enhancement for the copy post system, as for now you start to send a comment and it's just hold the post itself to copy the content (all or part of it), would be better if we have a real quoting system where I can click or hold the post while replying, a bottomsheet dialog appears like now but when you select "Copy Raw" instead of "copy all" you also have a "Quote" option that adds a > and all the text you selected previously. I know you can do it yourself, in fact it's just a small enhancement and I'm sure it doesn't require much work for it

**Comment by Wladefant** [comment]
I think the function should be called copy quote

**Reply by cris16228** [reply]
I'm not good with names but I guess it's good
